dashboard/app/controllers/scripts_controller.rb

```diff
@@ -1,9 +1,9 @@
 class ScriptsController < ApplicationController
   include VersionRedirectOverrider
 
-  before_action :require_levelbuilder_mode, except: [:show, :edit, :update]
+  before_action :require_levelbuilder_mode, except: [:show, :vocab, :resources, :code, :standards, :edit, :update]
```

Comment:
Why are these restricted to levelbuilder-mode? Is it just temporary until there's content on the page?

Reply:
They are the exception to requiring levelbuilder mode.

Reply:
🤦‍♀️ 🤦‍♀️ 🤦‍♀️ 🤦‍♀️